src/components/forms/tabs/RegisteredBoat.tsx

```diff
@@ -30,11 +29,10 @@ export default ({ prefix, ShowBigShipsForm, boatTypes }: Props) => (
         <FormattedMessage tagName="p" id="form.big_ship.text.inspection_and_insurance" />
       </div>
     )}
-    <FormattedMessage tagName="h3" id="form.registered.header.additional_info" />
-
     <BoatInfo />
-    <FormattedMessage tagName="h3" id="form.registered.header.accessibility" />
 
-    <Accessibility />
+    {mode === 'berth' && <Accessibility />}
```

Comment:
`berth` as constant pls, can be local constant, or under `/constants`

Reply:
Changed it, I was relying on TS :D